### PR TITLE
Change to match figma

### PR DESCRIPTION
**Pull request by PoohTheWinnie**

### CODE REVIEW

**Comment by PoohTheWinnie** [comment]
293026589_5359936807396313_8347764617600020497_n
Most recent commit

**Comment by PoohTheWinnie** [comment]
292511574_592365989067272_5578313345454808485_n

**Comment by PoohTheWinnie** [comment]
292606685_433438368656105_4256343128780368504_n

**Comment by PoohTheWinnie** [comment]
292597234_577397230458405_1395451363744231967_n

**Comment by PoohTheWinnie** [comment]
292597234_577397230458405_1395451363744231967_n

**Review by josephycyeh — COMMENTED** [review]
couple things, good work first of all on the ui. Some issues, if you check items, then add more items to cart, it messes it up, I think that is because of the indexes. Maybe we should use cartitem Ids to mark checked and unchecked instead of straight up index. try to do it, but I'm actually interested in solving this together in a pair programming session if you have trouble

File: screens/itemDetailScreen/itemDetailScreen.js [hunk 1]
```diff
@@ -225,6 +225,7 @@ export default function ItemDetailScreen({ navigation, route }) {
         </View>
       )}
       <View style={styles.safeContainer}>
+        <Text style={styles.boldMainText}>{item.name}</Text>
```
Comment (josephycyeh) [inline screens/itemDetailScreen/itemDetailScreen.js:228]: why is this here

Comment (PoohTheWinnie) [reply screens/itemDetailScreen/itemDetailScreen.js]: Accidental copy and paste, removed now

**Comment by josephycyeh** [comment]
we also want this behavior: when user presses select all, then add one more thing into cart, we want to unselect select all, but keep the ones already selected, does that make sense?

**Comment by PoohTheWinnie** [comment]
Select single item updates the state but is unable to update the checkmark on the button itself.  Works only when refreshing the page entirely
